.gitignore

```diff
@@ -49,3 +49,6 @@ node_modules
 
 # Ignore build for client
 /client/build
+
+# Ignore all yarn errors
+yarn-error.log
```

Comment:
Actually why there's a yarn log file in the main folder, should it be in the `client` folder instead?